### PR TITLE
Added notes on apparmor_parser install

### PR DESCRIPTION
**Pull request by jtravee**
Closes https://github.com/rancher/docs/issues/3651

Reference: https://docs.rke2.io/install/quickstart/#prerequisites

### CODE REVIEW

File: docs/install/quickstart.md [hunk 1]
```diff
@@ -8,9 +8,16 @@ This guide will help you quickly launch a cluster with default options.
...
 ### Prerequisites
 
-Make sure your environment fulfills the [requirements.](https://docs.rke2.io/install/requirements/)
+- Make sure your environment fulfills the [requirements.](https://docs.rke2.io/install/requirements/)
 If NetworkManager is installed and enabled on your hosts, [ensure that it is configured to ignore CNI-managed interfaces.](https://docs.rke2.io/known_issues/#networkmanager)
 
+- For RKE2 versions 1.21 and higher, if the host kernel has [AppArmor](https://apparmor.net/), the AppArmor userspace must be present also. You must install the apparmor_parser prior to the RKE2 installation:
+
+    ```
+    zypper install apparmor-parser
```
Comment (brandond) [inline docs/install/quickstart.md:17]: If we're going to document the SLES-specific command we should also document for Ubuntu, which is the other supported distro that ships with AppArmor. Alternately we could just leave it to users to know out how to install a package, which I think is fine.
```suggestion
    zypper install apparmor-parser
```

File: docs/install/quickstart.md [hunk 2]
```diff
@@ -8,9 +8,16 @@ This guide will help you quickly launch a cluster with default options.
 
 ### Prerequisites
 
-Make sure your environment fulfills the [requirements.](https://docs.rke2.io/install/requirements/)
+- Make sure your environment fulfills the [requirements.](https://docs.rke2.io/install/requirements/)
 If NetworkManager is installed and enabled on your hosts, [ensure that it is configured to ignore CNI-managed interfaces.](https://docs.rke2.io/known_issues/#networkmanager)
 
+- For RKE2 versions 1.21 and higher, if the host kernel has [AppArmor](https://apparmor.net/), the AppArmor userspace must be present also. You must install the apparmor_parser prior to the RKE2 installation:
```
Comment (brandond) [inline docs/install/quickstart.md:14]:
```suggestion
- For RKE2 versions 1.21 and higher, if the host kernel supports [AppArmor](https://apparmor.net/), the AppArmor tools (usually available via the `apparmor-parser` package) must also be present prior to installing RKE2.
```

**Review by brandond — CHANGES_REQUESTED** [review]

File: docs/install/quickstart.md (hunk 2 above)
Comment (jtravee) [reply docs/install/quickstart.md]: Revised as recommended.

**Review by brandond — APPROVED** [review]

**Comment by chrisbward** [comment]
https://we.riseup.net/wikis/300095#installing-apparmor-utilities

How does one enable aa in userland?

**Pull request merged**